backend/app/use_cases/generate.py

```diff
@@ -24,5 +24,18 @@ def execute(
         if output is None:
             output = []
 
+        new_output = []
+        for i in output:
+            new_output.append(
+                {
+                    "feature1": i.get_feature1(),
+                    "feature2": i.get_feature2(),
+                    "accuracy": 1 - i.get_accuracy(),
```

Comment:
Why are we subtracting by 1 here

Reply:
Previously, the output returned accuracy, but now we need to modify it to return bias instead, so we need to subtract by 1